modules/osdk-internal-crds.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-internal-crds_{context}"]
+= Internal or data-only objects
+
+It is a common practice for Operators to use Custom Resource Definitions (CRDs)
+internally to accomplish a task. These objects are not meant for users to
+manipulate and can be confusing to users of the Operator. For example, a
+database Operator might have a Replication CRD that is created whenever a user
+creates a Database object with `replication: true`.
+
+If a CRD is not meant for manipulation by users, it can be hidden by including
+its name within the `apps.operatorframework.io/internal-objects` annotation's
+array of values in the Operator's ClusterServiceVersion (CSV):
```

Comment:
Can you make this line a bit tighter?
_by including its name in the `apps.operatorframework.io/internal-objects` array in the Operator's ClusterServiceVersion (CSV)._